helpers/env.go

```diff
@@ -14,5 +14,5 @@ func EnvFallBack(envs []string, defaultValue string) string {
 
 // GetPowerEndPoint
 func GetPowerEndPoint() string {
-	return EnvFallBack([]string{"IBMCLOUD_POWER_API_ENDPOINT"}, "")
+	return EnvFallBack([]string{"IBMCLOUD_PI_API_ENDPOINT"}, "")
```

Comment:
I am afraid this will introduce regression for folks using `IBMCLOUD_POWER_API_ENDPOINT`. I know some projects using that.

Can we keep both and deprecate the use of `IBMCLOUD_POWER_API_ENDPOINT` with a warning message? We can then remove it completely in the next major release.

Reply:
I undid this change, as I believe we didn't actually need it to get terraform to run correctly.